test/map_node_test.cpp

```diff
@@ -0,0 +1,45 @@
+#include <gtest/gtest.h>
+#include <wl/default_map.h>
+
+namespace {
+const wl::MapGraph default_map = wl::default_map();
+} // namespace
+
+TEST(MapNode, neighbor_clockwise) {
```

Comment:
This line declares a test that will be run with `RUN_ALL_TESTS()` in the main function.

It will look like:
```
[ RUN      ] MapNode.neighbor_clockwise
```

Where the first name is the name of the **suite** of tests, and the second name describes the particular test in that suite.